content/blog/gatsby-tina-101.md

```diff
@@ -8,41 +8,41 @@ consumes:
+[TinaCMS](https://tinacms.org/docs/getting-started/introduction) is a _extensible_ toolkit that can help meet these needs. Tina allows developers to use formats we love, like Markdown and JSON, while providing a slick experience for content authors.
 
-I decided to add TinaCMS to an existing Gatsby site- my [Gatsby starter, _Tyra_](https://github.com/madelyneriksen/gatsby-starter-tyra/). Here's a walkthrough of my process that you can use to convert your own site!
+To understand better how Tina works, I decided to add it to an existing site, my [Gatsby starter, _Tyra_](https://github.com/madelyneriksen/gatsby-starter-tyra/). What follows is a walkthrough of my process. Feel free to use this as a reference for adding TinaCMS to an existing Gatsby site!
 
 > **In a rush?** _Check out the code [on Github](https://github.com/madelyneriksen/gatsby-starter-tyra)_
 
 ## Getting Started with Tina
 
-Tina is a non-intrusive library you can use to add editing capabilities to your site. All your page generation logic can remain _exactly_ the same- it's easy to take an existing Gatsby site and drop in dynamic content editing.
+Tina is a non-intrusive library you can use to add editing capabilities to your site. All your page generation logic can remain _exactly_ the same. This makes it easier to 'drop in' dynamic content editing.
 
-There's a collection of plugins you will need to install and register with Gatsby to add Tina into your site. Let's do that now.
+There are a collection of plugins you will need to install and register with Gatsby to add Tina into your site. Let's do that now.
 
 ### Installing Tina
 
-Like most things in the Javascript world, the packages we need are installable with `npm` or `yarn`. _Tyra_ uses NPM, but use whichever command is relevant for your project.
+Like most things in the Javascript world, we can install the packages we need with `npm` or `yarn`. Use whichever package manager is relevant for your project.
 
 ```bash
-# NPM
+# With npm
 npm i --save gatsby-plugin-tinacms gatsby-tinacms-git gatsby-tinacms-remark styled-components
 
 # Or using Yarn
 yarn add gatsby-plugin-tinacms gatsby-tinacms-git gatsby-tinacms-remark styled-components
 ```
 
-We need to add the Gatsby plugins for Tina itself, to interface Git with Tina, and to support markdown files (via Remark). Since _Tyra_ already had the Remark Gatsby plugin and all related dependencies installed, I only needed to install plugins specific to TinaCMS.
+This command adds the Gatsby plugins for Tina itself, to interface Git with Tina, and to support markdown files (via Remark). Since _Tyra_ already had the `gatsby-transformer-remark` plugin and all related dependencies installed, I only needed to install plugins specific to TinaCMS.
```

Comment:
```suggestion
This command adds the Gatsby plugins for Tina itself, to interface Git with Tina, and to support Markdown files (via Remark). Since _Tyra_ already had the `gatsby-transformer-remark` plugin and all related dependencies installed, I only needed to install plugins specific to TinaCMS.
```